enterprise-data-seeding/src/database.py

```
@@ -0,0 +1,165 @@
+"""
+Database connection management for seeding operations.
+
+Provides async and sync database session factories with proper
+connection pooling, transaction management, and error handling.
+"""
+
+from contextlib import asynccontextmanager, contextmanager
+from typing import AsyncGenerator, Generator
```

Ruff (0.16.2)

[warning] Import from `collections.abc` instead: `AsyncGenerator`, `Generator`

Import from `collections.abc`

(UP035)